framework/doc/content/source/fvics/FVFunctionIC.md

```diff
@@ -0,0 +1,17 @@
+# FVFunctionIC
```

Comment:
It might make sense to just have a `FVFunctorIC`

Reply:
you need dependency resolution then in case a variable has been passed

Reply:
Function works for me then to avoid possible issues